sunpy/timeseries/timeseriesbase.py

```diff
@@ -119,6 +119,15 @@ def __init__(self, data, meta=None, units=None, **kwargs):
 
 # #### Attribute definitions #### #
 
+    @property
+    def data(self):
+        warnings.warn("use `.to_dataframe` instead of `.data` to get the timeseries as a pandas dataframe.", SunpyUserWarning)
```

Comment:
```suggestion
        warnings.warn("use `.to_dataframe` instead of `.data` to get the timeseries as a pandas dataframe.",
                                SunpyUserWarning)
```